app/src/Navigators/Main.tsx

```diff
@@ -1,63 +1,123 @@
 /**
  * @file Defines the main screens that have tabs at the bottom of the app, e.g. Projects, Events.
  */
-
 import React from 'react'
-import { createBottomTabNavigator } from '@react-navigation/bottom-tabs'
+import {
+  createBottomTabNavigator,
+  BottomTabNavigationOptions,
+} from '@react-navigation/bottom-tabs'
+import { useRoute } from '@react-navigation/native'
 import { ListContainer } from '@/Containers'
-import Entypo from 'react-native-vector-icons/Entypo'
 import { ListType } from '@/Containers/ListContainer'
-import { useTheme } from '@/Hooks'
-import ProfileContainer from '@/NativeBase/Containers/ProfileContainer'
-import VerticalStackContainer from '@/NativeBase/Containers/VerticalStackContainer'
+import {
+  ProfileContainer,
+  VerticalStackContainer,
+} from '@/NativeBase/Containers'
+
+import SelectionIcons from '@/NativeBase/Assets/Icons/SelectionIcons'
+
+import { View, useColorMode } from 'native-base'
+import StaTheme from '@/NativeBase/Theme/StaTheme'
 
 const Tab = createBottomTabNavigator()
 
-// @refresh reset
+interface CustomNavBarIconProps extends BottomTabNavigationOptions {
+  focused?: boolean
+}
+
+const NavBarIcon = ({ focused }: CustomNavBarIconProps) => {
+  const { colorMode } = useColorMode()
+  const route = useRoute()
+  let iconName = route.name
+
+  switch (route.name) {
+    case 'Home':
+      iconName = 'home'
+      break
+    case 'Projects':
+      iconName = 'six-dots'
+      break
+    case 'Events':
+      iconName = 'ticket'
+      break
+    case 'Profile':
+      iconName = 'user'
+      break
+  }
+
+  return (
+    <View
+      borderTopColor={
+        focused
+          ? colorMode === 'light'
+            ? StaTheme.colors.primary['100']
+            : StaTheme.colors.primary['40']
+          : 'transparent'
+      }
+      borderTopWidth={2}
+      style={{
```

Comment:
It looks like the recommended way to do [width, height, etc](https://docs.nativebase.io/utility-props#h3-layout-width-and-height) and [flexbox properties](https://docs.nativebase.io/utility-props#h3-flexbox) is to use named utility props (`width=` etc) rather than one `style=` object?